README.md

```diff
@@ -1,2 +1,31 @@
 # theia-sprotty
 Glue code for sprotty diagrams in Theia
+
+
+### Quick start
+```bash
+yarn
+```
+
+### Building
+```bash
+yarn build
+```
+
+If you would like to run the `tsc` in a watch mode, run the following script:
+
+```bash
+yarn build -- -w
+```
+
+### Testing
```

Comment:
The only test is pretty bogus. We used it to spot some alleged DI issue, which turned out to be a missing pair of parentheses in our code. I'd rather delete it and not add the entire test infrastructure

Reply:
You mean deleting the test module itself?